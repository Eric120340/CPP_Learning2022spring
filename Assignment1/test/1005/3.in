4 4 896359
390724559 638858719 117618843 267245179 
596422602 24795548 783496619 170067010 
530712565 374096633 954222203 942017770 
30466077 17534236 915621900 427840191 
