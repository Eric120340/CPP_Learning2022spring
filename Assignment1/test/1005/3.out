368226917 254164976 742241511 579502748 
630941548 957329470 788791652 838925172 
322550544 438125733 740602695 946801738 
45254467 79535831 1503075 592321744 
